src/MainWindow.ui

```diff
@@ -866,7 +866,7 @@
     </widget>
     <widget class="QMenu" name="menuRemote">
      <property name="title">
-      <string>Remote</string>
+      <string>Re&amp;mote</string>
```

Comment:
Thanks, will look into those tomorrow or so! By the way, does anyone know how to prevent Qt Creator from adding keyboard shortcuts like these automatically?